client/src/components/messages/SendMessage.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { ImAttachment } from "react-icons/im";
+import { MdSend } from "react-icons/md";
+
+const SendMessage = () => {
+  return (
+    <div className="flex items-center w-full absolute bottom-0 overflow-x-hidden bg-slate-50 p-2">
+      <ImAttachment />
+      <form className="w-full flex bg-slate-50">
+        <input
+          type="text"
+          placeholder="Type a message"
+          className="w-full border-0 py-2 focus:outline-none bg-slate-50 px-2"
+        />
+        <button type="submit" className="inline-block w-12 ml-2">
+          <MdSend />
```

Comment:
If we pass this the prop  `color="#265B8B"` it'll take the primary color we've been using and make it more consistent with the overall color palette

Reply:
Are you referring to my use of "bg-slate-20" here?

Reply:
No to MdSend